FEATURE-LIFECYCLE-CHECKLIST.md

```diff
@@ -74,11 +71,10 @@ An Alpha feature can be officially labeled as Beta once it meets the following a
 A Beta feature can be officially labeled as Stable once it meets the following additional requirements:
 
 * Tests
-    * Test coverage is >= 90%
     * Automated tests are in place to prevent performance regressions.
 
 * Performance
-    * Latency, throughput and scale are quantified and documented.
+    * Latency, throughput and scalability are quantified and documented on istio.io.
 
 * Bugs
     * Feature has no outstanding P0 or P1 bugs.
```

Comment:
Sorry my original approval was that this was an incremental improvement not that I now agree with the whole doc. This doesn't make sense IMO. First of all, we have a lot of P1 bugs, that is quite a high bar. And what happens if a bug gets opened 1 day before we promote to stable, or 1 day after? Is it demoted?

Reply:
Title was poorly worded before. I cleaned it up. 

My thought is that there needs to be some sort of a bar. P0/P1 priority is generally applied to a release basis, and I think it makes sense to say that a beta/stable feature should not ship as part of a release if it has issues that the working group leads have marked as a p0 for that release. However, that's on a release level. 

I think there should be something that says that major issues are not present in that feature before it gets promoted though. We could say something along the lines of "Has been approved by appropriate workgroup and that group considers there to be no major issues at the time of promotion". That would leave the onus on the work group to decide if they consider it to be stable (functional without known issues?)

Reply:
That makes sense to me.